merlin/models/tf/prediction/classification.py

```diff
@@ -67,7 +73,7 @@ def _compute_loss(
         return self.loss(targets, predictions, sample_weight=sample_weight)
 
     def call(self, inputs, training=False, **kwargs):
-        return self.output_layer(inputs)
+        return self.output_activation(self.output_layer(inputs))
```

Comment:
Maybe we should set the activation function of the `output_later`  to `None` to avoid applying `sigmoid` two times?

Reply:
Good catch @sararb ! The activation of the dense later should actually be `linear` (no activation), as we are applying the `sigmoid` activation later to ensure it is tf.float32, following [TF documentation](https://www.tensorflow.org/guide/mixed_precision). Have just fixed it